docs/EN_US/ContainerizedHPCC/ContainerizedMods/AzureDeployChapter.xml

```diff
@@ -0,0 +1,645 @@
+        Systems pods. To check their status run:</para>
+
+        <programlisting>kubectl get pods </programlisting>
+
+        <variablelist>
+          <varlistentry>
+            <term>Note:</term>
+
+            <listitem>
+              <para>If this is the first time helm install has been run, it
+              will take some time for the pods to get to a Running state,
+              since Azure will need to pull the container images from Docker.
+              Once all the pods are running, the HPCC Systems Cluster is ready
+              to be used.</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+
+        <para> </para>
+      </sect3>
+    </sect2>
+
+    <sect2 id="Az-AccessECLWatch">
+      <title>Accessing ECLWatch</title>
+
+      <para>To access ECLWatch, an external IP to the ESP service running
+      ECLWatch is required. If you successfully deployed your cluster with the
+      proper visibility settings, then this will be listed as the
+      <emphasis>eclwatch</emphasis> service. The IP address can be obtained by
+      running the following command:</para>
+
+      <programlisting>kubectl get svc </programlisting>
+
+      <para>Your output should be similar to:</para>
+
+      <programlisting>NAME        TYPE         CLUSTER-IP    EXTERNAL-IP     PORT(S)         AGE
+eclservices ClusterIP    10.0.44.11    &lt;none&gt;          8010/TCP        11m
+<emphasis role="bold">eclwatch    LoadBalancer 10.0.21.16    12.87.156.228   8010:30190/TCP  11m</emphasis>
+kubernetes  ClusterIP    10.0.0.1      &lt;none&gt;          443/TCP         4h28m
+mydali      ClusterIP    10.0.195.229  &lt;none&gt;          7070/TCP        11m</programlisting>
+
+      <para>Use the EXTERNAL-IP address listed for the ECLWatch service. Open
+      a browser and go to http://&lt;external-ip&gt;:8010/. For example in
+      this case, go to http://12.87.156.228:8010. If everything is working as
+      expected, the ECLWatch landing page will be displayed.</para>
+    </sect2>
+
+    <sect2 id="AZUninstall_YourCluster">
+      <title>Uninstall Your Cluster</title>
+
+      <para>When you have completed using your HPCC Systems instance, unless
+      your project/job/work is intended to persist. It is a good idea to
+      remove all components deployed to the cloud.</para>
+
+      <para>In completely doing so you would remove the resource group you
+      created, which will irreversibly destroy any pods, contents, or work on
+      there, including all customizations. In addition if you have any jobs
+      that are in the ephemeral storage they will also be deleted. So before
+      undertaking this step carefully consider that this is permanent and make
+      certain that you really intend to remove everything. If unsure you might
+      better off to just leave it.</para>
+
+      <sect3>
+        <title>Stopping Your HPCC Systems Cluster</title>
```

Comment:
You can simply delete the resource group. It will destroy everything in it. Uninstalling  HPCC is not needed at all. In addition, uninstalling HPCC doesn't save you money. You will still be charged for the AKS.